deploy_website.sh

```diff
@@ -22,11 +22,19 @@
 # pip install mkdocs mkdocs-material
 # Preview the site as you're editing it with:
 # mkdocs serve
+#
+# Usage deploy_website.sh SHA_OR_REF_TO_DEPLOY
 
 set -ex
 
+# Get the actual SHA, even if a ref was passed.
+SHA=$(git rev-parse $1)
 REPO="git@github.com:square/workflow.git"
-DIR=temp-clone
+DIR=mkdocs-clone
```

Comment:
/tmp?

Reply:
You `cd ..` in this script v. `cd $ORIG_DIR` in the other one. Be good to be consistent, and again I'd suggest pushd / popd.

Reply:
Moved to pushd/popd.